### DISCUSSION TITLE
How to completely reinstall the router?

### DISCUSSION THREAD

**Discussion by lenovomen** [gnuton/asuswrt-merlin.ng#666] [Q&A]
Hi,

If someone buys a second hand router e.g. TUF 3000 V2, how can it be completely reinstalled with the bootloader?  How to reinstall like the iphone deep installer in DFU mode? Which rewrites it completely?

**Comment by gnuton** [comment]
You do not wanna touch the bootloader. There is no reason for it.
Just factory reset and flash the new firmware.
